.goreleaser.yml

```diff
@@ -14,39 +12,127 @@ builds:
+      - -X github.com/cosmos/cosmos-sdk/version.AppName=regen
+      - -X github.com/cosmos/cosmos-sdk/version.Version={{.Version}}
+      - -X github.com/cosmos/cosmos-sdk/version.Commit={{.Commit}}
+      - -X github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger,muslc
+      - -linkmode=external -extldflags "-L/home/runner/work -lwasmvm_muslc -Wl"
+
+#  - id: regen-darwin-arm64
+#    binary: regen
+#    main: ./app/regen
+#    goarch:
+#      - arm64
+#    goos:
+#      - darwin
+#    env:
+#      - LD_LIBRARY_PATH=/home/runner/work/osxcross/target/binaries/lib
+#    flags:
+#      - -mod=readonly
+#      - -tags="netgo ledger muslc"
+#      - -trimpath
+#    ldflags:
+#      - -s -w
+#      - -X github.com/cosmos/cosmos-sdk/version.Name=regen
+#      - -X github.com/cosmos/cosmos-sdk/version.AppName=regen
+#      - -X github.com/cosmos/cosmos-sdk/version.Version={{.Version}}
+#      - -X github.com/cosmos/cosmos-sdk/version.Commit={{.Commit}}
+#      - -X github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger,muslc
+#      - -linkmode=external -extldflags "-L/home/runner/work -lwasmvm_muslc -Wl"
 
   - id: regen-linux-amd64
     binary: regen
     main: ./app/regen
+    env:
+      - CGO_ENABLED=1
+      - CC=x86_64-linux-gnu-gcc
+      - CXX=x86_64-linux-gnu-g++
     goarch:
       - amd64
     goos:
       - linux
     flags:
-      - -tags="netgo ledger"
-      - -trimpath
       - -mod=readonly
+      - -tags="netgo ledger muslc"
+      - -trimpath
     ldflags:
-      - -s -w -X github.com/cosmos/cosmos-sdk/version.Name=regen -X github.com/cosmos/cosmos-sdk/version.AppName=regen -X github.com/cosmos/cosmos-sdk/version.Version={{.Version}} -X github.com/cosmos/cosmos-sdk/version.Commit={{.Commit}} -X "github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger"
- 
+      - -s -w
+      - -X github.com/cosmos/cosmos-sdk/version.Name=regen
+      - -X github.com/cosmos/cosmos-sdk/version.AppName=regen
+      - -X github.com/cosmos/cosmos-sdk/version.Version={{.Version}}
+      - -X github.com/cosmos/cosmos-sdk/version.Commit={{.Commit}}
+      - -X github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger,muslc
+      - -linkmode=external -extldflags "-L/home/runner/work -lwasmvm_muslc -Wl,-z,muldefs"
+
+#  - id: regen-linux-arm64
+#    binary: regen
+#    main: ./app/regen
+#    goarch:
+#      - arm64
+#    goos:
+#      - linux
+#    flags:
+#      - -mod=readonly
+#      - -tags="netgo ledger muslc"
+#      - -trimpath
+#    ldflags:
+#      - -s -w
+#      - -X github.com/cosmos/cosmos-sdk/version.Name=regen
+#      - -X github.com/cosmos/cosmos-sdk/version.AppName=regen
+#      - -X github.com/cosmos/cosmos-sdk/version.Version={{.Version}}
+#      - -X github.com/cosmos/cosmos-sdk/version.Commit={{.Commit}}
+#      - -X github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger,muslc
+#      - -linkmode=external -extldflags "-L/home/runner/work -lwasmvm_muslc -Wl,-z,muldefs"
+
+#  - id: regen-windows-amd64
```

Comment:
Windows also not supported by CosmWasm. Should be set up to work once we disable CosmWasm.